pkg/proxy/userspace/proxier.go

```diff
@@ -206,7 +207,10 @@ func NewCustomProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptab
 
 	err = setRLimit(64 * 1000)
 	if err != nil {
-		return nil, fmt.Errorf("failed to set open file handler limit: %v", err)
+		if !libcontainersystem.RunningInUserNS() {
```

Comment:
Why do we want this failure to return error if we are in the user namespace, but not if we aren't? It seems like, given the failure has happened, whether or not we are in the user ns should only impact the type of error message.

If we don't even want to attempt this operation if in the user ns, then we should perform the ns check earlier.

Reply:
> Why do we want this failure to return error if we are in the user namespace, but not if we aren't?

Maybe you have overlooked `!` symbol in the if condition?

> If we don't even want to attempt this operation if in the user ns

Worth trying even in userns, because the current process may already have > 64000 limit.

Reply:
it was mistake, fixed

Reply:
:+1: